apolloschurchapp/src/index.js

```diff
@@ -12,12 +13,14 @@ import { createNativeStackNavigator } from 'react-native-screens/native-stack';
 import SplashScreen from 'react-native-splash-screen';
 import 'react-native-gesture-handler'; // required for react-navigation
 import { enableScreens } from 'react-native-screens';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { BottomSheetModalProvider } from '@gorhom/bottom-sheet';
 
 import {
   BackgroundView,
-  withTheme,
   NavigationService,
-  Providers as ThemeProvider,
```

Comment:
I'm pretty sure we just need to add `Themer` to the[ list of providers](https://github.com/ApollosProject/apollos-apps/blob/master/packages/apollos-ui-kit/src/Providers.js#L8) that `ui-kit` exports. Doing it this way I think we'll lose some of the [other providers ](https://github.com/ApollosProject/apollos-apps/blob/master/packages/apollos-ui-kit/src/Providers.js#L8)in ui-kit?